module/config/default.conf

```diff
@@ -13,6 +13,7 @@ webinterface - "Webinterface":
 	bool activated : "Activated" = True
 	builtin;threaded;fastcgi;lightweight server : "Server" = builtin
 	bool https : "Use HTTPS" = False
+	bool dualstack : "Use Dualstack" = True
```

Comment:
Please change to `bool dualstack : "Listen on both IPv4 and IPv6 (IP must be set to 0.0.0.0)" = True`